docs/storage/snippets/tutorial/AspireStorage/AspireStorage.Worker/Properties/serviceDependencies.json

```diff
@@ -0,0 +1,7 @@
+{
+  "dependencies": {
```

Comment:
What is this?

Reply:
Deleted this too, it came from when I was trying to get the Azurite stuff working.